Detectors/TPC/reconstruction/src/TPCCATracking.cxx

```diff
@@ -286,6 +304,10 @@ int TPCCATracking::runTracking(const ClusterNativeAccessFullTPC& clusters, std::
       oTrack.setDeltaTBwd(tBwd);
       oTrack.setDeltaTFwd(tFwd);
       // RS: TODO: once the A/C merging will be implemented, both A and C side must be flagged for meged track
+      if (tracks[i].CCE()) {
+        oTrack.setHasCSideClusters();
+        oTrack.setHasASideClusters();
+      }
       if (tracks[i].CSide()) {
```

Comment:
should not it be ?
```
else {
  if (tracks[i].CSide()) {
     oTrack.setHasCSideClusters();
   } else {
      oTrack.setHasASideClusters();
   }
}
```

Reply:
does not make a difference in reality, but anyway, your version is cleaner so I changed it.